harness/helpers/helpers.ts

```diff
@@ -437,11 +441,21 @@ export class LocalTestClient extends BaseTestClient {
 
         return result
     }
+
+    getValidConfigPath() {
+        return `/client/${this.clientId}/config/v1/server/${this.sdkKey}.json`
+    }
+
+    getValidConfig() {
+        return this.options.enableRealtimeUpdates
+            ? config_sse(getMockServerUrl(), `/client/${this.clientId}/sse`)
```

Comment:
why not just always send the correct SSE paths in the config?

Reply:
Dunno - wanted this to be as minimal breaking as possible and that's what felt like an easy option/safe?

Reply:
the configs every SDK is pulling down are gonna have these SSE properties in them, so might as well make it consistent

Reply:
kk - will fix